app/src/main/java/com/kursivee/learn/common/api/MockyClient.kt

```diff
@@ -1,8 +1,17 @@
 package com.kursivee.learn.common.api
 
 import retrofit2.Retrofit
+import javax.inject.Inject
+import javax.inject.Singleton
 
-class MockyClient(var retrofit: Retrofit) {
+// We probably only want one instance of the MockyClient
```

Comment:
Yep I agree. I didn't have it marked as singleton in the commonmodule. Going to change that while I see pros/cons on implementation